<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chanelForm</class>
 <widget class="QWidget" name="chanelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>621</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Canal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="inputchanelFormcanal">
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>G</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Mínimo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputchanelFormMin">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Máximo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="inputchanelFormMax">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Esta função equaliza o histograma do canal selecionado. Use R,G, ou B para equalizar o canal Vermelho,Verde ou Azul,respectivamente.
Determine a faixa de intensidades a ser equalizada.
Para equalizar completamente o canal use 0 para o Mínimo e 255 para o Máximo.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
